packages/frontend-shared/src/locales/en-US.json

```diff
@@ -174,8 +174,8 @@
       "bodySuccess": "You are now logged in as {0}.",
       "bodyError": "An error occurred while attempting to log in:",
       "bodyBrowserError": "Cypress was unable to open your installed browser. To continue logging in, please paste this URL into your web browser:",
-      "dashboard": "Cypress Dashboard Service",
-      "titleInitial": "Log In To Cypress",
+      "cloud": "Cypress Cloud Service",
```

Comment:
Are we using "Cloud" in this release? I think this is still very much up in the air and may cause confusion if we haven't changed this verbiage throughout services before releasing this.